src/Evolution/Systems/ScalarWave/BoundaryConditions/ConstraintPreservingSphericalRadiation.cpp

```diff
@@ -62,22 +62,22 @@ void ConstraintPreservingSphericalRadiation<Dim>::pup(PUP::er& p) {
 template <size_t Dim>
 std::optional<std::string>
 ConstraintPreservingSphericalRadiation<Dim>::dg_time_derivative(
+    const gsl::not_null<Scalar<DataVector>*> dt_psi_correction,
+
```

Comment:
remove blank line when rebasing? Or maybe add it after the `not_null` arguments to add some separation and make things easier to read?